templates/dart/test/query_test.dart.twig

```diff
@@ -61,6 +68,14 @@ void main() {
           expect(query['method'], 'equal');
         });
       }
+
+      test('with a list', () {
+        final query = Query.equal('attr', ['a', 'b', 'c']).toJson();
```

Comment:
You can't call toJson() here.

Reply:
Because `Query.equal()` returns a `String` and `String`'s don't have a `toJson()` method.

Reply:
I created a String extension to handle it (line 22) 👍

Reply:
Adding the extension would make people question what it is and force people to go look at the implementation...why not just call `jsonDecode()`?

Reply:
Done